charms/kfp-api/tests/integration/test_charm.py

```diff
@@ -16,8 +16,32 @@
 APP_NAME = "kfp-api"
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 
-MINIO_CONFIG = {"access-key": "minio", "secret-key": "minio-secret-key"}
+KFP_DB = "kfp-db"
+KFP_DB_CHANNEL = "latest/edge"
 KFP_DB_CONFIG = {"database": "mlpipeline"}
+KFP_DB_ENTITY = "charmed-osm-mariadb-k8s"
```

Comment:
Could we instead say `MARIADB_CHARM` or soemthing similar. The thing is, `kfp-db` can be any of `mariadb` or `mysql`, to avoid confusion, I think this will be better.